app/views/hyrax/base/show.html.erb

```diff
@@ -6,32 +6,30 @@
 <%= render './shared/additional_citations' %>
 <div class="row work-type">
 <%# OVERRIDE Hyrax 2.9.1 here to hide the work type H1 title on all show-work pages %>
+  <div class="col-xs-12">&nbsp;</div>
```

Comment:
This file change looks like something for Hyku Prime; and is perhaps already accounted for.